custom-plugins/fictive_studios/admin/printing-area/create/print-area-create-view.php

```diff
@@ -0,0 +1,47 @@
+<form action="<?php echo esc_url(admin_url('admin-post.php')); ?>" method="post">
+  <?php if (isset($print_area_id)): ?>
+    <input type="hidden" name="id" value="<?php echo esc_attr($print_area_id); ?>">
+    <input type="hidden" name="action" value="edit_print_area">
+  <?php else: ?>
+    <input type="hidden" name="action" value="save_print_area">
+  <?php endif; ?>
+  <table class="form-table">
+    <tbody>
+      <tr>
+        <th scope="row">
+          <h2>Save Print Area</h2>
+        </th>
+      </tr>
+      <tr class="form-field">
+        <th scope="row"><label for="height">Height:</label></th>
+        <td><input type="number" id="height-field" name="height" value="<?php echo esc_attr($heightValue); ?>" required>
+        </td>
+      </tr>
+      <tr class="form-field">
+        <th scope="row"><label for="width">Width:</label></th>
+        <td><input type="number" id="width-field" name="width" value="<?php echo esc_attr($widthValue); ?>" required>
+        </td>
+      </tr>
+      <tr class="form-field">
+        <th scope="row"></th>
+        <td>
+          <button id="savePrintAreaBtn" type="submit" class="button button-primary">Submit</button>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</form>
```

Comment:
Removed the custom styling. Now uses default wordpress admin style format.
https://dotorgstyleguide.wordpress.com/